mne/io/snirf/_snirf.py

```diff
@@ -234,22 +245,35 @@ def natural_keys(text):
             elif "mm" in LengthUnit:
                 scal = 1000
 
+            srcPos3D /= scal
+            detPos3D /= scal
+
+            if optode_frame in ["head", "mri", "meg"]:
```

Comment:
@larsoner, I did this to mimic the behaviour in the nirx reader. Does the seem reasonable to you?

https://github.com/mne-tools/mne-python/blob/ddacc565962bab96d163e2cdb790304febf22582/mne/io/nirx/nirx.py#L238-L243

Reply:
Yes but I would use a tiny private function to do this to make the code DRY